include/seccomp-syscalls.h

```diff
@@ -280,6 +280,9 @@
 #define __PNR_atomic_barrier			-10246
 #define __PNR_atomic_cmpxchg_32			-10247
 #define __PNR_getpagesize			-10248
+#define __PNR_map_shadow_stack			-10249
```

Comment:
FTR: The PNR numbers below `-10244` (__PNR_memfd_secret) have already diverged between [`libseccomp:main`](https://github.com/seccomp/libseccomp/blob/main/include/seccomp-syscalls.h#L278) and [`libseccomp:release-2.5`](https://github.com/seccomp/libseccomp/blob/release-2.5/include/seccomp-syscalls.h#L278).

Reply:
As a FYI, I'm okay with fixing that sort of thing while merging the PR.

Reply:
I have preserved the `__PNR_map_shadow_stack` position while rebasing.